src/context.h

```diff
@@ -106,11 +103,7 @@ namespace librealsense
     class context : public std::enable_shared_from_this<context>
     {
     public:
-        explicit context(backend_type type,
-            const char* filename = nullptr,
-            const char* section = nullptr,
-            rs2_recording_mode mode = RS2_RECORDING_MODE_COUNT,
-            std::string min_api_version = "0.0.0");
+        explicit context( backend_type type );
```

Comment:
If you choose to keep this enum, maybe we should add as default value the unique value. Agree?

Reply:
No. On the 'dds` branch there's a default ctor, so this distinguishes it from another usage.

Reply:
So maybe there is an issue there, because it does not seem right that "standard" is not defaut, even if there are other possible values.
Let's talk about it tomorrow, please

Reply:
I want it to be explicit, just like it has been until now. You want this constructor, you specify `standard`.

Reply:
ok